vars/dockerExecuteOnKubernetes.groovy

```diff
@@ -416,6 +420,36 @@ private void unstashWorkspace(config, prefix) {
     }
 }
 
+private List getInitContainerList(config){
+    def initContainerSpecList = []
+    if (config.initContainerImage && config.containerMountPath) {
+        def initContainerName = config.initContainerImage.toLowerCase()
+        def initContainerSpec = [
+            name           : config.initContainerName,
+            image          : config.initContainerImage,
+            volumeMounts   : [[name: "volume", mountPath: config.containerMountPath]],
+        ]
+        if (config.initContainerCommand == null) {
+            echo "Please provide initContainerCommand for ${config.initContainerImage}"
+            initContainerSpec['command'] = [
+                '/usr/bin/tail',
+                '-f',
+                '/dev/null'
+            ]
+        } else {
+            echo "Command given : ${config.initContainerCommand}"
+            initContainerSpec['command'] = [
+                'sh',
```

Comment:
So whatever the command it will be run in a shell. Is this customary also with the other container commands?

Reply:
That is an interesting point here